uwreact_bringup/launch/zeds.launch

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0"?>
+
+<launch>
+    <!-- TODO(wraftus): Play around with scan_height and range_min -->
+    <arg name="scan_height"      default="10"/> 
+    <arg name="range_min"        default="0.45"/>
+    
+    <!-- TODO(wraftus): Make names of ZEDs more descriptive depending on location on robot-->
+    <!-- First ZED camera -->
+    <group ns="zed0">
+        <node name="zed_wrapper" pkg="zed_wrapper" type="zed_wrapper_node" output="screen" required="true">
```

Comment:
Should this be `required=true` or `respawn=true`? I suspect the latter but you would know better than I

Reply:
This is what's in their launch file so I just copied it over. Tbh not too sure what is best or even what the difference is

Reply:
`required=true` means that if the node crashes, it will pull down everything else in this launch file. `respawn=true` means that if the node crashes, it will automatically be relaunched. If neither are set, then the node can freely die without affecting the rest of the nodes in that launch file.

I think `respawn=true` is better for our application.